src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+
+  sourceString.split(';').forEach((line) => {
+    const trimmedLine = line.trim();
+
+    const [property, value] = trimmedLine.split(':');
+
+    if (property && value) {
+      resultObject[property.trim()] = value.trim().replace(/;/, '');
```

Comment:
The `replace` operation here is unnecessary because the `split` operation at line 11 already removes the semicolons at the end of each declaration. [CODE KNOWLEDGE]